### DISCUSSION TITLE
doris升级到1.2后建表失败，请大佬们看看是什么原因？

### DISCUSSION THREAD

**Discussion by JiaJunWu2022** [apache/doris#15726] [A - General / Q&A]
如下图所示，第一个建表语句跟第二个建表语句，区别只是最后一个字段的名称多了1个字符，但是第二个建表语句就是一直执行不成功，请问是什么原因导致的？是否是新版本对字段名称总长度做了什么限制？该如何解决？
![image](https://user-images.githubusercontent.com/121603105/211258191-631ed481-2a50-4ef4-b817-207723dcbf38.png)

建表语句如下：
`CREATE TABLE `TEST1` (
  `A` varchar(75) NOT NULL COMMENT '',
  `BBBBBBBB` datetime NOT NULL COMMENT '',
  `CCCCCCCC` varchar(50) NULL COMMENT '',
  `DDDDDDDD` datetime NULL COMMENT '',
  `EEEEEEEE` varchar(75) NOT NULL COMMENT '',
  `FFFFFFFF` varchar(75) NOT NULL COMMENT '',
  `GGGGGGGG` varchar(75) NULL COMMENT '',
  `HHHHHHHH` varchar(75) NULL COMMENT '',
  `IIIIIIII` varchar(75) NULL COMMENT '',
  `JJJJJJJJ` varchar(1500) NULL COMMENT '',
  `KKKKKKKK` int(11) NULL COMMENT '',
  `LLLLLLLL` varchar(15) NULL COMMENT '',
  `MMMMMMMM` varchar(15) NULL COMMENT '',
  `NNNNNNNN` varchar(150) NULL COMMENT '',
  `OOOOOOOO` varchar(15) NULL COMMENT '',
  `PPPPPPPP` varchar(105) NULL COMMENT '',
  `QQQQQQQQ` varchar(750) NULL COMMENT '',
  `RRRRRRRR` varchar(15) NULL COMMENT '',
  `SSSSSSSS` varchar(15) NULL COMMENT '',
  `TTTTTTTT` varchar(75) NULL COMMENT '',
  `UUUUUUUU` varchar(75) NULL COMMENT '',
  `VVVVVVVV` varchar(18) NULL COMMENT '',
  `WWWWWWWW` varchar(3) NULL COMMENT '',
  `XXXXXXXX` varchar(6000) NULL COMMENT '',
  `YYYYYYYYYYYYYYYYYYYYYYYY` varchar(150) NULL COMMENT '',
  `ZZZZZZZZZZZZZZZZZZZZZZZZZZZ` varchar(3) NULL COMMENT ''
) ENGINE=OLAP
UNIQUE KEY(`A`)
COMMENT '测试'
DISTRIBUTED BY HASH(`A`) BUCKETS 8
PROPERTIES (
"replication_allocation" = "tag.location.default: 1",
"in_memory" = "false",
"storage_format" = "V2",
"disable_auto_compaction" = "false"
);`

**Comment by JiaJunWu2022** [comment]
创建两个TEST1表不报错，是因为我执行完第一条SQL后将表drop掉了

**Comment by LemonLiTree** [comment]
看一下是不是fe 挂了？

**Reply by JiaJunWu2022** [reply]
没挂的，执行其他SQL都正常，就是建表失败

**Reply by LemonLiTree** [reply]
<img width="1240" alt="image" src="https://user-images.githubusercontent.com/109049295/211264053-a78d4143-c145-4c82-ad7f-9af33ab041fe.png">
我测试是ok的

**Reply by JiaJunWu2022** [reply]
你测试使用的版本是1.2.1吗？我在1.1.5版本测试也是ok的
